overlord/configstate/configcore/refresh.go

```diff
@@ -46,6 +48,16 @@ func validateRefreshSchedule(tr Conf) error {
 		}
 	}
 
+	refreshKeepInactiveStr, err := coreCfg(tr, "refresh.keep-inactive")
```

Comment:
How about "refresh.retain=N"?  This has a slightly more positive feeling to it, and also accounts for the total number instead of only the extras (logic needs to +1).

Reply:
only problem is that it needing to be >=2 is a bit weird

Reply:
OTOH, maybe it's good that it's weird: we should at some point revisit this and refactor things so you can indeed set it to 1. Right now it'd be too big a change for this little PR (the code that depends on this is nontrivial).